### ISSUE TITLE
SOHO-7686: Data Entered In Typeahead Not Available In The Dropdown Control

### ISSUE THREAD

**Issue by clepore** [infor-design/enterprise#253]
hi, I think we have an error in the Dropdown typeahead binding & I'm seeing the same thing in the 4.4.0 build.

The control I'm looking at is the Angular "Dropdown-Single Select" with "States - From Source".

What I'm looking for is a subscription to the typeahead data. Each time the user presses a key then my app needs to respond & do something extra. The way to do this is through the "source" property which the Angular control uses - [source]="source".

However, the source method (in dropdown.demo.ts) is getting called only once with a searchTerm of "".  It doesnt get called again after subsequent user keypresses.

Could you please have a look. Maybe this is expected behavior - if so could we please have a "searchTerm" Output event added to the API so we can subscribe that way?  Thanks

**Comment by EdwardCoyle** [comment]
Duplicate of #266.  Closing this one since we've got more info on the other

**Comment by brianjuan** [comment]
This is related to #267 which was passed QA testing.

**Comment by clepore** [comment]
Duplicate of #266
